src/backend/launcher.ts

```diff
@@ -254,13 +255,7 @@ async function prepareLaunch(
       }
     }
 
-    steamRuntime = [
-      path,
-      isNative || !gameInfo.install['install_path']
-        ? ''
-        : `--filesystem=${gameInfo.install['install_path']}`,
-      ...args
-    ]
```

Comment:
We are now setting `STEAM_INSTALL_PATH` which hints to the runtime what location should be mounted to container